website/hugo.toml

```diff
@@ -189,7 +188,7 @@ version = "v1.0"
 #  Set to true to disable breadcrumb navigation.
 breadcrumb_disable = false
 # Set to true to disable the About link in the site footer
```

Comment:
comment needs updation

Reply:
good catch! fixed in [ab9e6ca](https://github.com/siderolabs/talos/pull/10672/commits/ab9e6ca21aaf97083a5052289f46f7e270fef26e)